### PR TITLE
Overhaul software setup instructions

### LINKED ISSUE #114: Update virtual setup instructions
Setup instructions across OS's are somewhat stale, so let's refresh them! This issue focuses updated these two files:
-  [Windows instructions](https://github.com/AlexsLemonade/training-specific-template/blob/615e672511c8abf0e2ef92d62f98649851c5e2dd/virtual-setup/windows-instructions.md)
- [Mac instructions](https://github.com/AlexsLemonade/training-specific-template/blob/615e672511c8abf0e2ef92d62f98649851c5e2dd/virtual-setup/mac-instructions.md)

Notably, what I am proposing were not included in the overall planning note that folks looked at asynchronously, so feel free to weigh in here with any feedback @allyhawkins @Jen-OMalley @jaclyn-taroni @jashapiro !!

First, a statement about Mac screenshots in general - recent MacOS changes have really affected menus esp ~System Preferences~ System Settings. I think it probably suffices to leave screen shots and just state that different OS versions may have different menu names, but we can revisit these too - it shouldn't be _too_ much of a lift.
Edit - there may also be M vs intel chip considerations for screenshots (particularly for downloading), so this might be a place to look.

I can't speak to PC screenshots as I don't have a PC at the moment!

### Zoom instructions

I think at this point in society (vs March 2020..!), I think it is probably fine to get rid of Zoom _installation_ instructions throughout. But, we should still include instructions on setting Zoom preferences for Screen Share, particularly on the Mac side.

### Slack instructions

This should be somewhat modularized - specifically, "logging into the CCDS workspace" is content present in both  mac/windows. [This version](https://github.com/AlexsLemonade/training-specific-template/blob/615e672511c8abf0e2ef92d62f98649851c5e2dd/virtual-setup/mac-instructions.md?plain=1#L126-L151) of that content is most up-to-date, so let's move it into a separate markdown file that we can `%include%` in both Mac and PC.
Note that that the _header_ of this section should not be in the separate markdown, just the content, since we want to keep doctoc happy.

### LINKED ISSUE #123: Setup single HOME.md file
Once our remote vs. in-person config variable (see #106) is set up, we'll want to formally update the `HOME.md` file to handle in-person and remote. 

My original plan was to use jekyll include to bring in certain content from separate `home-<remote/inperson>.md` files into the main `HOME.md`, but we may not need to get that fancy! It could be fine here to just use `if` blocks and just directly write the text into each. 

for reference, here's some jekyll syntax - 

### `if`
```
# here page.platform is a page-specific variable but we'll want to use a "site" variable defined in config.
{% if page.platform == "mac" %}
Here's some info about the Mac.
{% elsif page.platform == "windows" %}
Here's some info about Windows ...
{% endif %}
```

### `include`

`{% include custom/mypage.html %}`

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #114 
Closed #123 

This PR dramatically simplifies setup instructions, and updates other files to match (this inlcuded `HOME.md`, so as a treat we get to close #123 too). I made various choices along the way; let me know what you think of them!

- I consolidated Windows and Mac installation instructions for Zoom and Slack into a single file `software-installation.md`, which replaces `software-setup.md` and `<os>-instructions.md`.
  - One motivation for doing this was there was a lot of repeated, but at time divergent!, text between mac and windows documents, so having one file seemed best.
  - I also wholesale removed a lot of the instructions for installing software, in particular Zoom. Unlike in March 2020, people know all about Zoom now! 
- I updated `HOME.md` to accommodate these changes, using `if` statements to control certain remote vs. in-person text.

I served the website with both `"remote"` and `"in-person"` flavors, and all formatting looks as I'd expect it to look.

After a first round of review where any overall organization/strategy can be discussed, I'll clean up the `software-setup/screenshots` directory to remove any that are no longer used.

### CODE REVIEW

File: software-setup/software-installation.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+One way that we'll use Zoom is via screen-sharing; it is important that you are able to share your screen with us so we can help you, as needed in 1:1 consulations sessions or in breakout rooms.
+
+If you are working on a Mac, you will need to grant Zoom permissions to be able to use screen share, as follows:
+
+{% comment %}
+Zoom's instructions are not yet updated for Ventura, but we could write instead:
+
+Follow [this link](https://support.zoom.us/hc/en-us/articles/360016688031) for full instructions on how to grant Zoom these permissions, if you have not previously done so.
+{% endcomment %}
```
Comment (sjspielman) [inline software-setup/software-installation.md:35]: Reviewers, have a look here. I was going to just include Zoom's own docs at first, but they aren't up-to-date, so I added the small subsections below.
Given that we do setup meetings for remote though, I think we can probably just link to Zoom's docs and get rid of the step-by-step instruction below. Thoughts?

File: software-setup/software-installation.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+We will use Slack to communicate with you both during and after the workshop, as described in our [Workshop Logistics](../workshop/workshop-logistics.md) and [Slack Procedures](slack-procedures.md) pages.
+
+
+### Installing Slack
+
+While Slack can be used directly from the browser at (<https://slack.com>), we recommend that you download Slack to be able to use its full features.
+Follow the download and installation instructions for your given operating system:
+
+* [Windows download link](https://slack.com/downloads/instructions/windows)
+* [macOS download link](https://slack.com/downloads/mac)
```
Comment (jashapiro) [inline software-setup/software-installation.md:65]: Linux download link? 
```suggestion
* [Linux download link](https://slack.com/downloads/linux)
```

File: software-setup/software-installation.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## Slack setup instructions
+
+We will use Slack to communicate with you both during and after the workshop, as described in our [Workshop Logistics](../workshop/workshop-logistics.md) and [Slack Procedures](slack-procedures.md) pages.
+
+
+### Installing Slack
+
+While Slack can be used directly from the browser at (<https://slack.com>), we recommend that you download Slack to be able to use its full features.
+Follow the download and installation instructions for your given operating system:
```
Comment (jashapiro) [inline software-setup/software-installation.md:62]: Alternatively, we can just send everyone to `https://slack.com/downloads`, which will redirect to the correct download for each system.

**Review by jashapiro — COMMENTED** [review]
Looks good overall. A few content thoughts.

File: software-setup/software-installation.md (hunk 3 above)
Comment (sjspielman) [reply software-setup/software-installation.md:62]: 👆

File: software-setup/software-installation.md [hunk 4]
```diff
@@ -24,7 +24,8 @@ If you do not have Zoom installed yet, you will need to download the Desktop cli
 
 ### Enabling screen share
 
-One way that we'll use Zoom is via screen-sharing; it is important that you are able to share your screen with us so we can help you, as needed in 1:1 consultations sessions or in breakout rooms.
+We will me making extensive use of Zoom screen sharing, both from our computers and yours.
```
Comment (jashapiro) [inline software-setup/software-installation.md:27]:
```suggestion
We will be making extensive use of Zoom screen sharing, both from our computers and yours.
```

File: software-setup/software-installation.md [hunk 5]
```diff
@@ -34,15 +35,15 @@ Zoom's instructions are not yet updated for Ventura, but we could write instead:
 Follow [this link](https://support.zoom.us/hc/en-us/articles/360016688031) for full instructions on how to grant Zoom these permissions, if you have not previously done so.
 {% endcomment %}
 
-#### For macOS Ventura and higher
+#### For macOS version 13 (Ventura) and higher
```
Comment (jashapiro) [inline software-setup/software-installation.md:38]: Some renderers are picky about whitespace after headers
```suggestion
#### For macOS version 13 (Ventura) and higher

```

File: software-setup/software-installation.md [hunk 6]
```diff
@@ -34,15 +35,15 @@ Zoom's instructions are not yet updated for Ventura, but we could write instead:
...
 * Open "System Settings"
 * Navigate to "Privacy & Security"
 * Click "Screen Recording"
 * Ensure the Zoom app is toggled on
   * If the Zoom app is not listed, click the plus sign to add Zoom to the list (you can find it in your Applications folder), and then toggle it on
 
 
-#### For macOS Monterey and lower
+#### For macOS version 12 (Monterey) and lower
```
Comment (jashapiro) [inline software-setup/software-installation.md:46]:
```suggestion
#### For macOS version 12 (Monterey) and lower

```

File: software-setup/software-installation.md [hunk 7]
```diff
@@ -59,11 +60,7 @@ We will use Slack to communicate with you both during and after the workshop, as
 ### Installing Slack
 
 While Slack can be used directly from the browser at (<https://slack.com>), we recommend that you download Slack to be able to use its full features.
-Follow the download and installation instructions for your given operating system:
-
-* [Windows download link](https://slack.com/downloads/instructions/windows)
-* [macOS download link](https://slack.com/downloads/mac)
-
+Download Slack from this link: <https://slack.com/downloads>, which will automatically direct you to the download page for your operating system.
```
Comment (jashapiro) [inline software-setup/software-installation.md:63]: And before headers.
```suggestion
Download Slack from this link: <https://slack.com/downloads>, which will automatically direct you to the download page for your operating system.

```

**Review by jashapiro — APPROVED** [review]
A few more minor updates.

**Pull request merged**